docs/fundamentals/week-1/learning-objectives.md

```diff
@@ -42,12 +42,6 @@ A full set of learning objectives are available on the [Lesson Plan](https://cod
 
 ## PD (introspection)
 
-- [ ] Explain the purpose of Personal Development
+- [ ] Explain the purpose of Professional Development
 - [ ] Believe in own capabilities and ability to be successful
```

Comment:
```suggestion
- [ ] List three personal capabilities you possess
```
So here, two things are conflated, and both are challenging to measure. It's also a huge demand. Can we demand people believe things, especially things without evidence. It might be an overreach for a learning objective.